rules/windows/sysmon/sysmon_outlook_newForm.yml

```diff
@@ -0,0 +1,25 @@
+title: Outlook Form Installation
+id: c3edc6a5-d9d4-48d8-930e-aab518390917
+status: experimental
+description: Detects the creation of new Outlook form which can contain malicious code
+references:
+    - https://twitter.com/blueteamsec1/status/1401290874202382336?s=20
+tags:
+    - attack.persistence
+    - attack.t1137.003
+author: Tobias Michalski
+date: 2021/06/10
+logsource:
+    product: windows
+    service: sysmon
```

Comment:
Why is this rule a sysmon rule and not a file_event? 

```
logsource:
    category: file_event
    product: windows
```

please use generic categories whenever possible.